packaging/aznfs/DEBIAN/postrm

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Exit on error.
+set -e
+
+chattr -i /opt/microsoft/aznfs/mountmap
+rm -r /opt/microsoft/aznfs
```

Comment:
as we discussed earlier, we do not control the filesystem of /opt on the user's system, so it may technically not support chattr, so we cannot use "set -e" here, remove "set -e" and check for rm status explicitly.
for chattr add -f to suppress warning 

chattr -if /opt/microsoft/aznfs/mountmap